llvm/test/TableGen/DAGISelEmitter-shouldignore.td

```diff
@@ -0,0 +1,37 @@
+// RUN: llvm-tblgen --gen-dag-isel -I %p/../../include %s -DIGNORE | FileCheck %s
+// RUN: llvm-tblgen --gen-dag-isel -I %p/../../include %s --match-prefix=CHECK_LOOK | FileCheck --check-prefix=CHECK_LOOK %s
```

Comment:
Should also generate globalisel and show that did import, and didn't produce the warning. The whole point is to not get warned on -warn-on-skipped-patterns

Reply:
Added this check to `test/TableGen/GlobalISelEmitterSkippedPatterns.td`.